en/02_Developer_Guides/15_Customising_the_Admin_Interface/06_Javascript_Development.md

```diff
@@ -132,71 +60,14 @@ Main advantages over simpler jQuery plugins are:
 *  Constructor/Destructor hooks
 *  Focus management and mouse interaction
 
-See the [official developer guide](http://jqueryui.com/docs/Developer_Guide) and other
-[tutorials](http://bililite.com/blog/understanding-jquery-ui-widgets-a-tutorial/) to get started.
-
-Example: Highlighter
-
-```js
-(function($) {
-  $.widget("ui.myHighlight", {
-    getBlink: function () {
-      return this._getData('blink');
-    },
-    setBlink: function (blink) {
-      this._setData('blink', blink);
-      if(blink) this.element.wrapInner('<blink></blink>');
-      else this.element.html(this.element.children().html());
-    },
-    _init: function() {
-      // grab the default value and use it
-      this.element.css('background',this.options.background);
-      this.element.css('color',this.options.foreground);
-      this.setBlink(this.options.blink);
-    }
-  });
-  // For demonstration purposes, this is also possible with jQuery.css()
-  $.ui.myHighlight.getter = "getBlink";
-  $.ui.myHighlight.defaults = {
-    foreground: "red",
-    background: "yellow",
-    blink: false
-  };
-})(jQuery);
-```
-
-Usage:
-
-
-```js
-(function($) {
-  // call with default options
-  $(':button').myHighlight();
-
-  // call with custom options
-  $(':button').myHighlight({background: "green"});
-
-  // set defaults for all future instances
-  $.ui.myHighlight.defaults.background = "green";
-
-  // Adjust property after initialization
-  $(':button').myHighlight('setBlink', true);
-
-  // Get property
-  $(':button').myHighlight('getBlink');
-})(jQuery);
-```
+See the [official API documentation](https://api.jqueryui.com/) and read up about the
+[jQuery UI Widget Factory](https://learn.jquery.com/jquery-ui/widget-factory/) to get started.
 
 ### jQuery.Entwine
 
-jQuery.entwine is a third-party plugin, from its documentation:
-"A basic desire for jQuery programming is some sort of OO or other organisational method for code. For your
```

Comment:
I couldn't find that quote - but we're including the entwine docs directly now anyway so it doesn't make sense to quote them like this.